docs/docsite/rst/dev_guide/testing/sanity/no-underscore-variable.rst

```diff
@@ -0,0 +1,28 @@
+Sanity Tests » no-underscore-variable
+=====================================
+
+In the future, Ansible may use the identifier ``_`` to internationalize its
+message strings.  To be ready for that, we need to make sure that there are
+no conflicting identifiers defined in the code base.
+
+In common practice, ``_`` is frequently used as a dummy variable (a variable
+to receive a value from a function where the value is useless and never used).
+In Ansible, we're using the identifier ``dummy`` for this purpose instead.
```

Comment:
I tested the following with pylint 1.7.2:
```
def arigato():
    return None, None, 'test'

def roboto():
    dummy, dummy, msg = arigato()
    filtered_stuff = [misc_thing for misc_thing, dummy in ((1, 2), (3, 4))]
    print(msg)
    return filtered_stuff


if __name__ == '__main__':
    roboto()
```

Here's the pylint messages:
```
$ pylint test.py                                        *[code-smell-_]  (21:47:51)
************* Module abadger.test
C0111:  1, 0: Missing module docstring (missing-docstring)
C0111:  1, 0: Missing function docstring (missing-docstring)
C0111:  4, 0: Missing function docstring (missing-docstring)
```

So it looks like repeated use of dummy is okay.